core-doc-element-page.html

```diff
@@ -0,0 +1,127 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../paper-styles/paper-styles.html">
+
+<!--
+Renders documentation describing an element's API.
+
+`core-doc-element-page` renders element descriptions as extracted by
+[Hydrolysis](https://github.com/PolymerLabs/hydrolysis). You can provide them
+either via binding (the {@link #model} attribute)...
+
+    <core-doc-page model="{{elementDescription}}"></core-doc-page>
+
+...or by placing the element description in JSON as the text content of a
+`core-doc-page`:
+
+    <core-doc-page>
+      {
+        "is": "awesome-sauce",
+        "properties": [
+          {"name": "isAwesome", "type": "boolean", "desc": "Is it awesome?"},
+        ]
+      }
+    </core-doc-page>
+
+-->
+<dom-module id="core-doc-element-page">
+
+  <link rel="stylesheet" href="core-doc-element-page.css">
+
+  <template>
+
+    <header paper-font="display2">&lt;<span>{{model.is}}</span>&gt;</header>
```

Comment:
Yeah, too bad CSS doesn't let us mix in styles (yet) :P  I'm hoping that the new styling solution replaces this, and it just becomes a `@mixin(--paper-font-display2)` in your CSS